Framework/RemoteAlgorithms/src/AbortRemoteJob.cpp

```diff
@@ -33,7 +33,6 @@ void AbortRemoteJob::init() {
   // Unlike most algorithms, this one doesn't deal with workspaces....
 
   auto requireValue = boost::make_shared<MandatoryValidator<std::string>>();
-  auto nullValidator = boost::make_shared<NullValidator>();
```

Comment:
Can we delete the Nullvalidator headers?

Reply:
Yes, we can. Thanks for spotting this!